src/db_adapters/accounts.rs

```diff
@@ -141,3 +141,62 @@ async fn remove_accounts(
         }
     }
 }
+
+pub(crate) async fn store_accounts_from_genesis(near_config: near_indexer::NearConfig) {
+    info!(
+        target: crate::INDEXER_FOR_EXPLORER,
+        "Adding/updating accounts from genesis..."
+    );
+    let pool = crate::models::establish_connection();
+
+    let accounts_models: Vec<models::accounts::Account> = near_config
+        .genesis
+        .records
+        .0
+        .iter()
+        .filter_map(|record| {
+            if let near_indexer::near_primitives::state_record::StateRecord::Account {
+                account_id,
+                ..
+            } = record
+            {
+                Some(models::accounts::Account::new(account_id.to_string(), None))
+            } else {
+                None
+            }
+        })
+        .collect();
+
+    loop {
+        match diesel::insert_into(schema::accounts::table)
+            .values(accounts_models.clone())
```

Comment:
There are 213596 accounts in testnet genesis. Let's split this into batches as we do in Indexer for Wallet, I guess.